lib/companies_web/views/company_view.ex

```diff
@@ -16,6 +16,10 @@ defmodule CompaniesWeb.CompanyView do
     |> String.replace("www.", "")
   end
 
+  def url_with_scheme("http://" <> _ = url), do: url
+  def url_with_scheme("https://" <> _ = url), do: url
+  def url_with_scheme(url, scheme \\ "http"), do: "#{scheme}://#{url}"
```

Comment:
I think we want the default to be `https`, what do you think?

Reply:
Even one or the other might lead to issues (like a url not been set properly to redirect to one or the other), we maybe want to do something more clever on the long run. It's fine for now in https I guess

Reply:
I can look into the cloudflare setup to ensure http goes to https.

Reply:
But that's the thing, one might have setup their domain to only use one of them (I know, I have done this shit already :blush)

Reply:
how about trying to retrieve the endpoint from here?
https://github.com/doomspork/elixir-companies/blob/2872b679c40b468e7ae4c01e0d6c34deb4bc7545/config/prod.exs#L14

Reply:
I'm thinking default scheme to https and open another issue for adding scheme required validation for company & job url, what's your opinion?

Reply:
@gonglexin sounds good to me 👍